ambry-store/src/main/java/com.github.ambry.store/DiskManager.java

```diff
@@ -379,4 +381,27 @@ boolean areAllStoresDown() {
     }
     return true;
   }
+
+  /**
+   * Reports any unrecognized directories on disk
+   */
+  private void reportUnrecognizedDirs() {
+    File[] dirs = new File(disk.getMountPath()).listFiles(File::isDirectory);
```

Comment:
I have a quick question: looks like we only report the unexpected directories. Do we need to report unexpected files?